src/main.rs

```diff
@@ -26,7 +26,7 @@ fn main() {
 }
 
 // Make a separate runner to print errors using Display instead of Debug
-fn try_main() -> Result<Void, Error> {
+fn try_main() -> Result<(), Error> {
```

Comment:
I can't remember the reason for using `Void`, but thanks for getting rid of that dependency! Could you also fix the unused import and the `Cargo.toml` file?

Reply:
Before we used `exec` in the sense that it "took over" the current process to run the new command. That way instead of using pipes etc. for stdin and stdout, the started command (such as `gdb`) would be executed in place of the `cargo with` command. Not doing this might have unfortunate consequences when using interactive commands, hence we have to make sure that this PR doesn't break that usecase.

Reply:
that's a very good point. I could leave this function alone, possibly, and execute in a different manner with one of the below options.